config/initializers/honeycomb.rb

```diff
@@ -0,0 +1,14 @@
+Honeycomb.configure do |config|
+  config.write_key = ENV.fetch('HONEYCOMB_WRITEKEY', 'd5529d94d4e989bb1fc1b1cfc559ebba')
+  config.dataset = ENV.fetch('HONEYCOMB_DATASET', 'od2-rails-staging')
```

Comment:
If this is environment, is it vaguely leaking our honeycomb config? Should the default focus for local development? This might be a non-issue.

Reply:
yeah we can make the defaults be something different as long as it can pass CCI tests, or we can not configure it for test